app/src/main/res/layout/activity_privacy_policy.xml

```diff
@@ -16,46 +16,32 @@
   along with this program.  If not, see <https://www.gnu.org/licenses/>.
   -->
 
-<android.support.constraint.ConstraintLayout
```

Comment:
Is there a reason why replaced it with a linear layout?

Reply:
Easier, better performance and less memory consumption for such an easy layout.